v2/devices/yggdrasil.yml

```diff
@@ -194,111 +194,91 @@ operating_systems:
+                  group: "firmware"
+                - partition: "lk2"
+                  file: "unpacked/lk-verified.img"
+                  group: "firmware"
+                - partition: "sspm_2"
+                  file: "unpacked/sspm-verified.img"
+                  group: "firmware"
+                - partition: "tee2"
+                  file: "unpacked/tee-verified.img"
+                  group: "firmware"
+                - partition: "lk"
+                  file: "unpacked/lk-verified.img"
+                  group: "firmware"
+                - partition: "sspm_1"
+                  file: "unpacked/sspm-verified.img"
+                  group: "firmware"
+                - partition: "tee1"
+                  file: "unpacked/tee-verified.img"
+                  group: "firmware"
+                - partition: "preloader"
+                  file: "unpacked/preloader_k63v2_64_bsp.bin"
+                  group: "firmware"
+                - partition: "system"
+                  file: "unpacked/system.img"
+                  group: "firmware"
+                - partition: "vendor"
+                  file: "unpacked/vendor.img"
                   group: "firmware"
-        condition:
-          var: "install_twrp"
-          value: true
-      - actions:
-          - core:user_action:
-              action: "recovery"
-        condition:
-          var: "install_twrp"
-          value: true
-      - actions:
-          - core:user_action:
-              action: "twrp_unlock"
-      - actions:
-          - core:info:
-              status: "Restoring NVDATA partition"
-              dots: true
-              progress: 0.1
-              info: Creating directory
-          - adb:shell:
-              args: ["mkdir", "/mnt/vendor/nvdata"]
-          - core:info:
-              progress: 0.4
-              info: Mounting partition
-          - adb:shell:
-              args: ["mount", "/dev/block/by-name/nvdata", "/mnt/vendor/nvdata"]
-          - core:info:
-              progress: 0.8
-              info: Restoring permissions
-          - adb:shell:
-              args:
-                [
-                  "find",
-                  "/mnt/vendor/nvdata",
-                  "-type",
-                  "f",
-                  "-exec",
-                  "restorecon",
-                  "-v",
-                  "{}",
-                  "+"
-                ]
-          - core:info:
-              progress: 0.9
-              info: Unmounting partition
-          - adb:shell:
-              args: ["umount", "/mnt/vendor/nvdata"]
-          - core:info:
-              progress: 0
       - actions:
-          - core:user_action:
-              action: "twrp_sideload"
+          - fastboot:erase:
+              partition: "userdata"
       - actions:
-          - adb:sideload:
-              group: "VollaOS"
-              file: "volla-9.0-20201019-nightly-k63v2_64_bsp-signed.zip"
+          - fastboot:erase:
```

Comment:
erase, not format? Does Android format it to ext4 correctly on boot?

Reply:
yes